vhdbuilder/packer/cleanup.sh

```diff
@@ -44,6 +44,28 @@ if [ -n "$id" ]; then
   az storage account delete -n ${SA_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} --yes
 fi
 
+#clean up storage account created over X hours ago (testing; X=19)
+# set subscription to AKS VHD Image Release
+az account set -s a15c116e-99e3-4c59-aebc-8f864929b4a0
```

Comment:
I think that we do not need to set account. If we must set it, it should also impact all old commands after this line.

Reply:
> I think that we do not need to set account. If we must set it, it should also impact all old commands after this line.

Yes. It was set in Line 7, init-variables.sh. Checked the log of "Build Windows VHD" step. It was already set to this subscription.